src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -206,13 +209,17 @@ internal ModelParameters(IReadOnlyList<IReadOnlyList<WordItemScore>> wordScoresP
             }
         }
 
-        [BestFriend]
-        internal ModelParameters GetLdaDetails(int iinfo)
+        /// <summary>
+        /// Method to provide details about the topic discovered by LightLDA
+        /// </summary>
+        /// <param name="topicIndex">index of topic</param>
+        /// <returns></returns>
+        public ModelParameters GetLdaDetails(int topicIndex)
         {
-            Contracts.Assert(0 <= iinfo && iinfo < _ldas.Length);
+            Contracts.Assert(0 <= topicIndex && topicIndex < _ldas.Length);
 
-            var ldaState = _ldas[iinfo];
-            var mapping = _columnMappings[iinfo];
+            var ldaState = _ldas[topicIndex];
+            var mapping = _columnMappings[topicIndex];
 
             return ldaState.GetLdaSummary(mapping);
```

Comment:
The way I understand it `GetLdaDetails(i)` will give you the LdaSummary for _column_ `i` not the _topic_ `i` (and each column will have several topics). So I think that renaming `iinfo` to `topicIndex` and the documentation you added is somewhat misguiding.

Reply:
You are right! I confused this with topic index, thanks

---
In reply to: [445984832](https://github.com/dotnet/machinelearning/pull/5260#discussion_r445984832) [](ancestors = 445984832)